tox.ini

```diff
@@ -0,0 +1,88 @@
+[tox]
+min_version = 4
+# list of environments to run by default with tox run
+env_list =
+    lint,
+    compile,
+    py312-docs,
+    py3132-tests,
+    a11y-tests,
+
+# general tox env configuration
+# these can be run with any py3{9,12} version
+# tox run -e py39-lint  
+[testenv]
+deps =
+    lint: pre-commit
+    compile: sphinx-theme-builder[cli]
+    profile-docs: py-spy
+extras = 
+    {docs-no-checks, docs-linkcheck, profile-docs}: doc
+skip_install = 
+    lint: true # do not need to install to lint
+    ; {compile,docs-no-checks, docs-linkcheck}: false # default but just in case
+commands = 
+    lint: pre-commit run -a
+    compile: stb compile
+    # can substitute the target directory
+    docs-no-checks: sphinx-build {posargs:audit}/site {posargs:audit}/_build
+    docs-linkcheck: sphinx-build -W -b linkcheck docs/ docs/_build/html --keep-going
```

Comment:
Added a linkcheck task to check for broken links in our docs - not hooked to CI or anything atm